CExts.c

```diff
@@ -0,0 +1,59 @@
+//
```

Comment:
Could you remove the headers from the file to be consistent with the rest of the codebase?

Reply:
Sure